tests/integration/test_repos_repo.py

```diff
@@ -29,8 +29,18 @@ def test_assignees(self):
             for assignee in repository.assignees():
                 assert isinstance(assignee, github3.users.User)
 
+    def test_branch(self):
+        """Test the ability to retrieve a single branch in a repository."""
+        cassette_name = self.cassette_name('branch')
+        with self.recorder.use_cassette(cassette_name):
+            repository = self.gh.repository('sigmavirus24', 'github3.py')
+            assert repository is not None
+            branch = repository.branch('develop')
+            assert isinstance(branch, github3.repos.branch.Branch)
+            assert branch.protection['enabled']
```

Comment:
Great feedback. Updated to be more explicit regarding the presence of the enabled field as part of the protection attribute. I don't actually care what it's value is, and don't want the test to be tied to an external configuration setting.